src/Form/Type/MissionTypeAssetTypesType.php

```diff
@@ -16,15 +16,15 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
     {
         $builder
             ->add('type', ChoiceType::class, [
-                'label' => 'Type',
+                'label' => 'common.type',
```

Comment:
I would recommend to keep translations keys unique to prevent a too generic translation being updated everywhere:
```suggestion
                'label' => 'missionType.assetTypes.type',
```
WDYT @mRoca ?